src/topics/inputs.md

```diff
@@ -143,13 +143,85 @@ The `baseCommand` field will always appear in the final command line before the
 
 [touch]: http://www.linfo.org/touch.html
 
+### Optional Input Parameters
+
+Optional `inputs` include `label` and `secondaryFiles`.
+`label` is a short, human-readable description for the parameter.
+
+```cwl
+inputs:
+  example_file:
+    type: File
+    label: Use case for label parameter
+    inputBinding:
+      position: 1
+```
+
+`secondaryFiles` is an optional input parameter that provides a pattern of specifying files
+or directories that must be included alongside the primary file. 
+The following example demonstrates the `secondaryFiles` input parameter'
+
+```cwl
+inputs:
+  example_file:
+    type: file
+    inputBinding:
+      position: 1
+      prefix: --file=
+      seperate: false
+    "secondaryFiles": [
```

Comment:
Are the double quotes necessary? I normally use them when I need to use a dash in a string in YAML. Not sure if it's needed here.